business/layer.go

```diff
@@ -51,15 +52,16 @@ func NewWithBackends(k8s kubernetes.IstioClientInterface, prom prometheus.Client
 	temporaryLayer.Health = HealthService{prom: prom, k8s: k8s}
 	temporaryLayer.Svc = SvcService{prom: prom, k8s: k8s, businessLayer: temporaryLayer}
 	temporaryLayer.IstioConfig = IstioConfigService{k8s: k8s}
-	temporaryLayer.Workload = WorkloadService{k8s: k8s, prom: prom, ns: NewNamespaceService(k8s)}
 	temporaryLayer.Validations = IstioValidationsService{k8s: k8s, ws: temporaryLayer.Workload}
 	temporaryLayer.App = AppService{k8s: k8s}
 	temporaryLayer.Namespace = NewNamespaceService(k8s)
+	temporaryLayer.Workload = WorkloadService{k8s: k8s, prom: prom, businessLayer: temporaryLayer}
```

Comment:
Problems started right after changing this line. It makes [business/workloads.go#L215](https://github.com/kiali/kiali/pull/684/commits/dc3cb0a239776ad16154ebba6fef4e9fbded3e28#diff-9b809ab418e2f384025bbe22c950eb75R215) to access to k8s (which it has a nil value).

```log
  | I1128 12:27:24.461122       1 workloads.go:214] k8s var is present? %!(EXTRA bool=false, <nil>)
```